src/query/xasl.h

```diff
@@ -176,6 +188,25 @@ typedef enum
   CTE_PROC
 } PROC_TYPE;
 
+typedef struct qproc_db_value_list *QPROC_DB_VALUE_LIST;	/* TODO */
+struct qproc_db_value_list
+{
+  QPROC_DB_VALUE_LIST next;
+  DB_VALUE *val;
+  TP_DOMAIN *dom;
+
+    qproc_db_value_list () = default;
```

Comment:
This file seems to some indentation issues (mainly declaration of constructors of structs).

Reply:
fixed the indentation issues.